Source/DafnyStandardLibraries/src/DafnyStdLibs/Arithmetic/Internal/DivInternalsNonlinear.dfy

```diff
@@ -6,7 +6,14 @@
  *  SPDX-License-Identifier: MIT 
  *******************************************************************************/
 
-module DafnyStdLibs.Arithmetic.DivInternalsNonlinear {
+/**
+Several lemmas in this module are bodyless and marked as axiom.
+These lemmas could previously be proven with an empty body, but only if this file was verified by itself,
+not with the rest of the standard library.
+If we want to remove the {:axiom} annotation from these lemmas, 
+we will have to investigate why they only verify under specific conditions.   
+*/
```

Comment:
This comment is obsolete now. You could comment on the arithmetic solver setting, though.